secator/utils_test.py

```
@@ -239,7 +239,11 @@ def mock_command(cls, inputs=[], opts={}, fixture=None, method=''):
 	else:
 		mocks.append(fixture)
 
-	with mock_subprocess_popen(mocks):
+	# Mocked commands never actually run or install, so treat them as installed —
+	# otherwise the runner's presence check (shutil.which on the fake `cmd`) fails
+	# and triggers auto-install during the test.
+	with mock_subprocess_popen(mocks), \
+			unittest.mock.patch.object(cls, 'is_installed', return_value=True):
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)